### PR TITLE
Add Longhorn 1.3.0 chart

### PR DESCRIPTION
**Pull request by innobead**
Signed-off-by: David Ko <dko@suse.com>

cc @yangchiu @cchien816 @kaxing

### CODE REVIEW

File: charts/longhorn/100.2.1+up1.3.0-rc3/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+annotations:
+  catalog.cattle.io/auto-install: longhorn-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Longhorn
+  catalog.cattle.io/kube-version: '>=1.18.0-0'
```
Comment (PennyScissors) [inline charts/longhorn/100.2.1+up1.3.0-rc3/Chart.yaml:5]: Please update this upstream to the format `'>= 1.18.0-0 < 1.XX.0-0'` where XX is the max Kubernetes version the chart support +1. More on the reasoning here: https://github.com/rancher/charts/pull/1929

File: charts/longhorn/100.2.1+up1.3.0-rc3/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  catalog.cattle.io/namespace: longhorn-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/provides-gvr: longhorn.io/v1beta1
+  catalog.cattle.io/rancher-version: '>= 2.6.0-0 <= 2.6.100-0'
```
Comment (PennyScissors) [inline charts/longhorn/100.2.1+up1.3.0-rc3/Chart.yaml:9]: Update this to `'>= 2.6.0-0 < 2.7.0-0'`

File: charts/longhorn/100.2.1+up1.3.0-rc3/Chart.yaml (hunk 1 above)
Comment (innobead) [reply charts/longhorn/100.2.1+up1.3.0-rc3/Chart.yaml]: Resolved. cc @PhanLe1010

File: charts/longhorn/100.2.1+up1.3.0-rc3/Chart.yaml (hunk 2 above)
Comment (innobead) [reply charts/longhorn/100.2.1+up1.3.0-rc3/Chart.yaml]: Resolved.

**Comment by cchien816** [comment]
**Verified on https://github.com/innobead/charts-1.git branch longhorn-1.3.0-2.6
Result Pass**

**Precondition**
   - Setup Rancher v2.6.5
   - Navigation to Rancher -> Apps -> Repositories -> Create
   - Add repo by `https://github.com/innobead/charts-1.git` branch `longhorn-1.3.0-2.6`

**Steps**
1. Fresh install  
   - Navigation to Rancher -> Apps -> Charts, select new appeared Longhorn charts
   - Install v1.3.0 (use image from longhornio)
   - Create volume/pvc/deployment and write data
   - Scale down deployment then scale up, verify pod data kept consistent
   - Uninstall

2. Upgrade
   - Navigation to Rancher -> Apps -> Charts, select new appeared Longhorn charts
   - Install v1.2.4
   - Create volume/pvc/deployment and write data
   - Use chart upgrade Longhorn to v1.3.0
   - Upgrade engine image
   - Verify pod data kept consistent
   - Uninstall

cc @yangchiu

**Review by cchien816 — APPROVED** [review]
Use PR code base verified install / upgrade from v1.2.4 on Rancher pass.
Detail steps logged [here](https://github.com/rancher/charts/pull/1899#issuecomment-1157387809)

**Review by shuo-wu — DISMISSED** [review]

**Review by derekbit — DISMISSED** [review]

**Comment by iosifnicolae2** [comment]
Looking forward for this PR to be merged :)

**Comment by innobead** [comment]
@PennyScissors Please help merge this. Thanks.

**Comment by PennyScissors** [comment]
LGTM overall, my only concern is that we have never released a longhorn version `100.2.0+upX.X.X` so you skipped that version and used `100.2.1+up1.3.0`. Was that intentional or should this chart have version `100.2.0+up1.3.0` instead?

This is what we have released in 2.6 so far for reference:
image

This also needs to be rebased, but please feel free to message me on Slack after and I'll approve asap!

**Comment by innobead** [comment]
> Use PR code base verified install / upgrade from v1.2.4 on Rancher pass. Detail steps logged [here](https://github.com/rancher/charts/pull/1899#issuecomment-1157387809)

@PennyScissors rebased and change the version to `100.2.0+up1.3.0` instead. 

@cchien816 Just rebased the code from dev-v2.6, so please help verify this chart PR again to see if all good before merging.

**Review by cchien816 — APPROVED** [review]
**Verified by https://github.com/innobead/charts-1/tree/longhorn-1.3.0-2.6
Result Pass**

Test [steps](https://github.com/rancher/charts/pull/1899#issuecomment-1157387809)

**Review by shuo-wu — APPROVED** [review]

**Review by PennyScissors — DISMISSED** [review]

**Comment by innobead** [comment]
@PennyScissors Rebased again. Please help approve and merge. Thanks.

**Review by PennyScissors — APPROVED** [review]

**Comment by iosifnicolae2** [comment]
The v1.3.0 images haven't been pushed to Docker repository so we're getting ` ErrImagePull` error.
https://hub.docker.com/r/rancher/longhornio-longhorn-ui/tags
```bash
docker pull rancher/longhornio-longhorn-ui:v1.3.0
```
```
Error response from daemon: manifest for rancher/longhornio-longhorn-ui:v1.3.0 not found: manifest unknown: manifest unknown
```

**Pull request merged**